### ISSUE TITLE
Frequency of prediction

### ISSUE THREAD

**Issue by eugenelyj** [daniilidis-group/EV-FlowNet#6]
Thanks for your sharing of this work!! I have tested on my PC and also read the paper, it seems that the frequency of optical flow prediction is the same as the RGB frame, right? If so, have you compared it with the work using RGB frame?

**Comment by alexzzhu** [comment]
Yes, you are correct. We report numbers comparing this method to UnFlow, which is a self-supervised method trained on images, applied to the grayscale frames from the DAVIS.

**Comment by Jee-King** [comment]
@eugenelyj  @alexzzhu 
Hi, I read this paper. it seems that at training time, the network is provided with the event timestamp images, as well as a pair of grayscale images for calculating optical flow.  However, why  this test code only uses gray images as input(the test dataset _outdoor_1_ only contains gray images )? 
Wainting for your apply. Thanks!

**Comment by alexzzhu** [comment]
@Jee-King I'm not sure where you are seeing that the test code only uses grayscale images. In the forward pass of the network, here: https://github.com/daniilidis-group/EV-FlowNet/blob/cf801ef5e4b112e08bfe6eac4f0878b1f5aac2fe/src/test.py#L33, the network only sees the event images.
